include/seqan3/search/configuration/max_error.hpp

```diff
@@ -89,9 +89,34 @@ class max_error_insertion : public pipeable_config_element<max_error_insertion,
  * ### Example
  * \include test/snippet/search/configuration_error.cpp
  */
-class max_error_deletion : public pipeable_config_element<max_error_deletion, std::variant<error_count, error_rate>>
+struct max_error_deletion : public pipeable_config_element<max_error_deletion>
 {
-public:
+    //!\brief  The number of deletion errors allowed for the search.
```

Comment:
```suggestion
    //!\brief  The number of deletion errors or the deletion error rate, allowed for the search.
```

Reply:
I'd propose: `The error count or error rate.` That it is meant for the search is already clear by the class's documentation.